src/dc_receive_imf.rs

```diff
@@ -1085,27 +1081,26 @@ unsafe fn create_or_lookup_group(
                         );
                         if !optional_field.is_null() {
                             X_MrGrpImageChanged = (*optional_field).fld_value;
-                            mime_parser.is_system_message = 3;
-                            better_msg = dc_stock_system_msg(
-                                context,
+                            mime_parser.is_system_message = 3i32;
+                            better_msg = context.stock_system_msg(
                                 if strcmp(
                                     X_MrGrpImageChanged,
                                     b"0\x00" as *const u8 as *const libc::c_char,
                                 ) == 0
```

Comment:
sidenote: is this not the same as strlen(X_MrGrpImageChanged) == 0?

Reply:
Afaik we're comparing `X_MrGrpImageChanged` with the string `"0"` which isn't the same thing.

Reply:
ups, true!